tests/foreman/cli/test_location.py

```diff
@@ -45,8 +35,10 @@ def test_create_location_with_different_names_positive(self, name):
         @Assert: Location is created successfully and has proper name
 
         """
-        loc = make_location({'name': name})
-        self.assertEqual(loc['name'], name)
+        for name in valid_data_list():
```

Comment:
@oshtaier I am not sure if we need to fix the tests.  This should have been written as a bug.  Did somebody create a bug? If you can give the bug number, I can skip that specific test based on bugzilla.